include/common/debug.h

```diff
@@ -47,5 +51,7 @@ noreturn void panic(const char *fmt, ...) __printf(1, 2);
 #else
 #define debug(...) ((void)0)
 #endif
-
```

Comment:
Please keep the blank line before the final `#endif`

Reply:
done